### ISSUE TITLE
InjectConfig: automatic toArray

### ISSUE THREAD

**Issue by AnnaDamm** [reinfi/zf-dependency-injection#38]
Hi,

when using `@InjectConfig` with an array of strings/integers, currently you still get a Config-Object. It would be nice to have a way to do an automatic toArray() conversion. Maybe via parameter on the Annotation?

```php
// current code
/**
 * @InjectConfig('redis.hosts')
 * @param Config $hosts
 */
__construct(Config $hosts) {
    $this->hosts = $hosts->toArray();
}
// suggested code
/**
 * @InjectConfig('redis.hosts', asArray=true)
 * @param string[] $hosts
 */
__construct(array $hosts) {
    $this->hosts = $hosts;
}
```

**Comment by reinfi** [comment]
Good idea, will work on it as soon as possible.

**Comment by reinfi** [comment]
Released with either 3.6.3 for Zend oder 4.0.2 for Laminas.
